### PR TITLE
[DEV-6902] Adds parent_toptier_agency_id to federal_account endpoint

### PR DESCRIPTION
**Pull request by collinwr**
**Description:**
Adds new field `parent_agency_toptier_code` and `parent_agency_name` to the `/api/v2/federal_accounts/{account_number}/` endpoint. 

**Technical details:**
Simple change to the `values` selection in a Django query. Removed some unrelated Object Definitions from API Contract. 

**Requirements for PR merge:**

1. [x] Unit & integration tests updated
2. [x] API documentation updated
3. [ ] Necessary PR reviewers:
    - [ ] Backend
    - [x] Frontend <OPTIONAL>
4. [x] Matview impact assessment completed
5. [x] Frontend impact assessment completed
6. [x] Data validation completed
7. [x] Appropriate Operations ticket(s) created
8. [x] Jira Ticket [DEV-6902](https://federal-spending-transparency.atlassian.net/browse/DEV-6902):
    - [x] Link to this Pull-Request
    - [ ] Performance evaluation of affected (API | Script | Download)
    - [ ] Before / After data comparison

**Area for explaining above N/A when needed:**
```
```

### CODE REVIEW

File: usaspending_api/api_contracts/contracts/v2/federal_accounts/account_number.md
Comment (ebdabbs) [inline usaspending_api/api_contracts/contracts/v2/federal_accounts/account_number.md:23]: If this is a number, we should remove `` around 126

File: usaspending_api/api_contracts/contracts/v2/federal_accounts/account_number.md
Comment (ebdabbs) [inline usaspending_api/api_contracts/contracts/v2/federal_accounts/account_number.md:24]: Thanks for cleaning up!

**Review by ebdabbs — CHANGES_REQUESTED** [review]

File: usaspending_api/api_contracts/contracts/v2/federal_accounts/account_number.md
Comment (sethstoudenmier) [inline usaspending_api/api_contracts/contracts/v2/federal_accounts/account_number.md:23]: Think this should be (required, string) for both new items.

The toptier_code has numbers but is stored as a text type because those numbers can lead with zeros. To that point, if `126` is just an arbitrary number I would recommend putting placeholder with a leading zero so that it is more clear.

Comment (collinwr) [reply usaspending_api/api_contracts/contracts/v2/federal_accounts/account_number.md]: Good call. Resolved

**Review by sethstoudenmier — APPROVED** [review]
Changes LGTM. Will still need code owner approval since it touches API contracts.

**Review by ebdabbs — APPROVED** [review]
API contract LGTM!

**Pull request merged**
